### PR TITLE
Delete instance normalization from discriminator conv part

### LINKED ISSUE #30: Instance normalization is used only in generator's downsampling
Seeing current implementation of this repository,  discriminator has instance normalization, but it is against architecture on the paper.

According to the paper,
> We base our generator network G(yi(t), eˆi; ψ, P) on the
image-to-image translation architecture proposed by Johnson et. al. [20], but replace downsampling and upsampling layers with residual blocks similarly to [6] (with batch normalization [16] replaced by instance normalization [38])

> ional part of the discriminator V (xi(t), yi(t); θ), we use similar networks, which consist of residual downsampling blocks (same as the ones used in the generator, but without normalization layers)

### PR DESCRIPTION
**Pull request by asterisk37n**
Close #30 

- Remove two instance normalization in the last residual block in conv part in discriminator

I mind if pretrained weights might not be loaded after the change of architecture.

### CODE REVIEW

**Comment by vincent-thevenin** [comment]
I modified my shared pretrained weight, green light for the merge
